kofta/src/vscode-webview/types.ts

```diff
@@ -25,6 +25,7 @@ export type User = {
   currentRoom?: Room;
   followsYou?: boolean;
   youAreFollowing?: boolean;
+  setshouldPlayChatNotificationSound: boolean;
```

Comment:
`setshouldPlayChatNotificationSound` -> `setShouldPlayChatNotificationSound`

Reply:
I didn't get this one, what do you mean?